fix:Update CONTRIBUTING.md (#375)
meteroid-oss/meteroid: CONTRIBUTING.md

> - Start the database with docker compose. If you intend to run the Metering app as well, you will need the "metering"
>   profile as follows:
>   `docker compose -f docker/develop/docker-compose.yml --env-file .env --profile metering up`
> - If docker display
>   `meteroid-db  |  FATAL:  role "meteroid" does not exist`
>   You can check your docker version and update

Reviewer: ```suggestion
- Start the database with :   
`docker compose -f docker/develop/docker-compose.yml --env-file .env`. 

If you intend to run the Metering app as well, you will need to use the "metering"
  profile instead :
  `docker compose -f docker/develop/docker-compose.yml --env-file .env --profile metering up`
```

this would solve it for future users no matter the docker version, and prevent discrepancies between what is used in docker compose & what is used in the backend

Author: sorry I'm having a hard time reproducing it.But actually on top of my old docker and docker composer, `sudo docker compose -f docker/develop/docker-compose.yml --env-file .env config`display right env config 
```
meteroid-db:
    command:
      - postgres
      - -c
      - wal_level=logical
    container_name: meteroid-db
    environment:
      POSTGRES_DB: meteroid
      POSTGRES_PASSWORD: secret
      POSTGRES_USER: meteroid
    healthcheck:
      test:
        - CMD-SHELL
        - pg_isready -d $${POSTGRES_DB} -U $${POSTGRES_USER}
      timeout: 5s
      interval: 10s
      retries: 5
    image: postgres:15.2
    networks:
      meteroid_net: null
    ports:
      - mode: ingress
        target: 5432
        published: "5432"
        protocol: tcp
    user: postgres
    volumes:
      - type: volume
        source: pg_data
        target: /var/lib/postgresql/data
        volume: {}
```
As you said
> I would guess that either you have that env set at a higher level/in your PATH, or for some reason the default in `POSTGRES_USER=${DATABASE_USER:-meteroid}` was not interpolated.
>

Reviewer: The interpolation works here in the old docker version ( `POSTGRES_DB: meteroid` ) so it's not that.

I really don't understand how the docker version could impact that specific postgres container env resolution (even more if it's just from 2022)

We can include it in the "Version requirements" section of the contributing file (` - Docker >= 27 `), + the update I suggested above. 
If that's fine with you let's settle with that for this PR ! I'd rather not explore specific bugs in the Contributing guide to avoid bloating the file, it's important that new users follow it.

(at some point we may add some "Common_issues.md" file to list common errors & resolutions, for issues covered in discord or hard to find with github search)

Author: LMGT.
